### ISSUE TITLE
Seperators in backend breadcrumb too low after contao 4.9.12 update

### ISSUE THREAD

**Issue by bohnmedia** [contao/contao#2769]
After the contao 4.9.12 update the seperators are too low in backend breadcrumb.

Chrome 88, Windows 10
![image](https://user-images.githubusercontent.com/9419148/108222951-90b83d00-7139-11eb-8147-2029ae74484c.png)

Firefox 85, Windows 10
![image](https://user-images.githubusercontent.com/9419148/108223066-b2192900-7139-11eb-8350-d79728326216.png)

Edge 88, Windows 10
![image](https://user-images.githubusercontent.com/9419148/108223173-d37a1500-7139-11eb-8eab-2ee41e0cb939.png)

You should remove line-height from "#main_headline span", give it "vertical-align:top" and assign the line height to "#main_headline".

```stylesheet
#main_headline {
    margin: 18px 0;
    padding: 0 16px;
    font-size: 1.1rem;
    line-height: 22px;
}

#main_headline span {
    display: inline-block;
    max-width: 30%;
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
    vertical-align: top;
}
```

Or you can use this pull request. https://github.com/contao/contao/pull/2751

**Comment by bohnmedia** [comment]
Issue: https://github.com/contao/contao/issues/2686

**Comment by fritzmg** [comment]
The `line-height` is still too low in other parts. Example from the site structure:

<img src="https://user-images.githubusercontent.com/4970961/108224078-6dd65a80-7132-11eb-8c86-8c52cabf6e72.png" width="74.4">

(This is supposed to be `gg, qq`).

The problem is that the back end CSS sets `line-height: 1` globally. However, for `overflow: hidden` to work without these issues, the `line-height` needs to be `normal`.

**Comment by bohnmedia** [comment]
A global solution seems to be the better way to fix this problem. I will create a new pull request for this in the next days.

**Comment by leofeyer** [comment]
Let's discuss possible solutions before we start adding pull requests, so we can be sure that the issue is fixed for every browser on every platform.

**Comment by bohnmedia** [comment]
Ok, I think it would make sense to set line-height globally to normal as suggested by fritz and reduce the paddings to have nearly the same spacings like before relative to the containers. Text with line breaks would be a litte higher than before.

**Comment by leofeyer** [comment]
`line-height: normal` will change every margin and padding all throughout the back end.

### Current

<img width="1130" alt="" src="https://user-images.githubusercontent.com/1192057/108228280-ffe46000-713e-11eb-8553-c6ab25268537.png">

### New

<img width="1130" alt="" src="https://user-images.githubusercontent.com/1192057/108228319-08d53180-713f-11eb-81ad-cf78179473b7.png">

So if this turns out to be a Firefox-only problem, I would prefer to fix this in Firefox only. Otherwise we will have to adjust a whole lot of spacings. 🙈

**Comment by fritzmg** [comment]
> Ok, I think it would make sense to set line-height globally to normal as suggested by fritz

Well, my suggestions was not really to change the `line-height` globally, as that would probably be a lot of work, since you would have to change a lot of things then. imho it is just something we have to live with for now. In order to fix the issue itself, we could just adjust the elements that were set to `overflow: hidden` recently. i.e. change their `line-height` to `normal` and then adjust any paddings or margins.

**Comment by leofeyer** [comment]
That sounds more reasonable. 👍🏻

**Comment by fritzmg** [comment]
> So if this turns out to be a Firefox-only problem, I would prefer to fix this in Firefox only.

Not sure why this is Firefox-only. If you set the `line-height` to `1` or `1em` you are _reducing_ the normal line-height and thus cutting off any character that would otherwise fit into the normal line-height:

<img src="https://user-images.githubusercontent.com/4970961/108229128-6e252480-7137-11eb-8e7c-c0ca490de216.png" width="480">

imho that it works in other browsers is only by chance, somehow.

**Comment by fritzmg** [comment]
It looks the same in Google Chrome btw., so it is not Browser related. If anything it is OS related, as font rendering is quite different between MacOS and Windows.

**Comment by bohnmedia** [comment]
For a quick fix maybe better, but a line-height of 1 does not feel right. ;) I will check the effort this evening.

**Comment by bohnmedia** [comment]
I checked the css and could change everything to a normal line-height.

There are two ways of adjustment.
- Pixelperfect adjustments with some more code
- Adjustments of approximately 1 pixel per element with the same amount or less code

If it is ok for you I would like to modify the css with the second solution.

**Comment by leofeyer** [comment]
I cannot say if solution no. 2 is ok for me. I need to see the CSS or at least you need to elaborate on the difference.

**Comment by Aybee** [comment]
Basically, it is not a good idea to set all elements to _line-height:1_, which is achieved by `body{line-height:1}`. It is recommended (e.g. here https://www.mediaevent.de/css/line-height.html, https://developer.mozilla.org/en-US/docs/Web/CSS/line-height) to leave the line-height at normal, depending on the browser that would be between 1.2 and 1.4, or to set it to 1.5.

If you then come across a situation where a line-height of 1 is required, which actually never occurs, you can then explicitly set the line-height there to 1.

That's why I recommend revising the CSS again with `body{line-height:1.5}`. You can probably also save a lot on paddings, which were certainly placed here and there.

**Comment by leofeyer** [comment]
Interestingly enough, using `line-height: normal` does not even fix the issue on Firefox 86 on macOS. However, the following does:

```css
#main_headline span {
  display: inline-flex; /* previously inline-block */
}
```

Can you confirm?

**Comment by m-vo** [comment]
Just tested: `display: inline-flex;` also fixes it for Firefox on Ubuntu. 

IMHO a more stable approach in these scenarios is making the wrapper `display: flex; align-items: baseline;` and then readding horizontal gap/margins between the elements. Inline positioning is a fiddly thing.
